### DISCUSSION TITLE
[New features] Read new and/or receipt messages

### DISCUSSION THREAD

**Discussion by Tharnum** [dimaskiddo/go-whatsapp-multidevice-rest#55] [Ideas]
Hi! I've found your project very helpful for my needs. However, I need reassurance that my sent messages are delivered successfully, as this is crucial for my use case. I'm curious if you plan to implement these features or this project in maintenance mode?

**Comment by Tharnum** [comment]
Sad... (

**Comment by dimaskiddo** [comment]
Hai @Tharnum currently it's must be successfull delivered when the API return success status because it will check the number is it registered or not an returning message id as sign that message with that message id is successfully sent
